Tests/js/apps/plugins/assets/ez-updatetreeplugin-tests.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-updatetreeplugin-tests', function (Y) {
+    var registerTest, eventTest,
+        Mock = Y.Mock;
+
+    eventTest = new Y.Test.Case({
+        name: 'eZ Tree update events tests',
+
+        setUp: function () {
+            var App;
+
+            this.discoveryBarInstanceMock = new Mock();
+            this.treeActionViewMock = new Mock();
+            this.treeMock = new Mock();
+            Y.Mock.expect(this.treeMock, {
+                method: 'clear',
+                callCount: 1,
+            });
+            Y.Mock.expect(this.treeActionViewMock, {
+                method: 'get',
+                args: ['tree'],
+                returns: this.treeMock
+            });
+            Y.Mock.expect(this.discoveryBarInstanceMock, {
+                method: 'getAction',
+                args: ['tree'],
+                returns: this.treeActionViewMock
+            });
+
+            App = Y.Base.create('testApp', Y.Base, [], {
+                sideViews: {
+                    discoveryBar: {
+                        instance : this.discoveryBarInstanceMock
+                    },
+                }
+            }, {});
+
+            this.app = new App();
+            this.plugin = new Y.eZ.Plugin.UpdateTree({
+                host: this.app,
+            });
+        },
+
+        tearDown: function () {
+            this.plugin.destroy();
+            delete this.plugin;
+            this.app.destroy();
+            delete this.app;
+        },
+
+        clearTreeOnEvent: function (eventName) {
```

Comment:
should be called _clearTreeOnEvent and _doNotClearTreeOnEvent so that those methods are executed as tests by YUITest.